src/js/components/SideBar/SideBarTabOptionsDate.vue

```diff
@@ -40,7 +40,11 @@
 			<div>
 				<div class="selectUnit">
 					<input v-model="move.step">
-					<Multiselect v-model="move.unit" :options="move.units" />
+					<Multiselect
+						v-model="move.unit"
+						:options="move.units"
+						label="name"
+						track-by="name" />
```

Comment:
I'd personally use track-by="value" as they are the keys and name is just the rendered label. In this case it doesn't really matter, because they should both be unique.

Reply:
done